Makefile

```diff
@@ -157,4 +161,7 @@ e2e-cleanup:
 	helm uninstall e2e -n oam-system
 	kubectl delete namespace oam-system --wait
 
-e2e: e2e-setup e2e-test go-integration
\ No newline at end of file
+e2e: e2e-setup e2e-test go-integration
+
+prepare-legacy-chart:
+	rsync -r $(LEGACY_HELM_CHARTS_DIR)/$(LEGACY_HELM_CHARTS) $(HELM_CHARTS_DIR)
```

Comment:
Can you explain why we need this step? Could the legacy Helm chart instead just live in `$HELM_CHARTS_DIR` to begin with?

Reply:
We intended to store legacy chart in a separate directory as most users won't care about the legacy one. It would be easier to put it in `$HELM_CHARTS_DIR`. @wonderflow